HolyshiftService/Program.cs

```diff
@@ -1,24 +1,81 @@
-﻿using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.Configuration;
-using Microsoft.EntityFrameworkCore;
-using Pomelo.EntityFrameworkCore.MySql.Infrastructure;
-using System;
-using Microsoft.Extensions.Options;
+using Microsoft.AspNetCore.Authentication.JwtBearer;
+using System.Text;
+using Microsoft.IdentityModel.Tokens;
+
 
 var builder = WebApplication.CreateBuilder(args);
 
+var configDirectory = Path.Combine(Directory.GetCurrentDirectory(), "Config");
+
+builder.Configuration
+    .SetBasePath(configDirectory) 
+    .AddJsonFile("appsettings.json", optional: false, reloadOnChange: true);
+

```

Comment:
Why? I don't see any modification in `appsettings.json` or it's location and it should already be working and automatically added.

I believe this is unintended and should be reverted.

Reply:
after we changed the structure in the last pr I started having problems because the appsettings.json isn't used no matter where I put it ,  so that's how I solved it

Reply:
Let's pair on it, because I am sure you don't need this line. It might be the folder restructuring broke it some how.